js/server.js

```diff
@@ -19,7 +19,7 @@ class Connection {
   constructor(ws) {
     this.ws = ws
     this.name = null
-    console.log(`hello ${this.address()}`)
+    console.logVerbose(`hello ${this.address()}`)
```

Comment:
I only see one log statement there

Andrew Johnson
Founder
*gaiagps.com <http://gaiagps.com>*

On Sat, Jun 13, 2015 at 2:05 PM, Kevin Lacker <notifications@github.com>
wrote:

> In js/server.js
> <https://github.com/lacker/cardkit/pull/14#discussion_r32373867>:
>
> > @@ -19,7 +19,7 @@ class Connection {
> >    constructor(ws) {
> >      this.ws = ws
> >      this.name = null
> > -    console.log(`hello ${this.address()}`)
> > +    console.logVerbose(`hello ${this.address()}`)
>
> How does this work? It seems like this would not work.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/lacker/cardkit/pull/14/files#r32373867>.
>